### ISSUE TITLE
Cannot find include files with relative paths on Windows

### ISSUE THREAD

**Issue by amai2012** [danmar/simplecpp#80]
This ticket was transferred from http://trac.cppcheck.net/ticket/8093 "cppcheck cannot find include files with relative paths on Windows"
Verified by a local cygwin/mingw build based on c61b868a970b878f673f029a1e3e81dbe81a652d on Windows.
Further details, see trac.

**Comment by amai2012** [comment]
Output on Windows:
````
c:\test\build\Win32\VC110>simplecpp ..\..\..\src\MapFile\MapFileReaderBase.cpp

../src/MapFile/MapFileReaderBase.h:4: missing header: Header not found: "MapFileError.h"
``

**Comment by amai2012** [comment]
That might be addressed by #82 ...
